documentation/modules/ROOT/pages/installing-imagebaseinstall-cluster.adoc

```diff
@@ -579,7 +579,7 @@ configMapGenerator:
     - hub-1/extra-manifest/enable-crun-master.yaml
     - hub-1/extra-manifest/enable-crun-worker.yaml
     name: ran-rds-418-manifests
-    namespace: sno-ibi
+    namespace: sno-abi
```

Comment:
This doc is for IBI, are we sure we need to use ABI namespace?

Reply:
Yes, it needs to remain the same for the configmap generator.